### ISSUE TITLE
Test Failure: io.openliberty.microprofile.metrics.internal.fat.DefaultHistogramBucketsTest.testDefaultBucketsEnabledHistogram

### ISSUE THREAD

**Issue by fmhwong** [OpenLiberty/open-liberty#26904]
RTC 297838

```
testDefaultBucketsEnabledHistogram:java.net.ConnectException: 2023-11-10-02:32:17:008 Connection refused
    at java.base/sun.nio.ch.Net.connect0(Native Method)
    at java.base/sun.nio.ch.Net.connect(Net.java:579)
    at java.base/sun.nio.ch.Net.connect(Net.java:568)
    at java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:588)
    at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:327)
    at java.base/java.net.Socket.connect(Socket.java:633)
    at java.base/sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:304)
    at java.base/sun.security.ssl.BaseSSLSocketImpl.connect(BaseSSLSocketImpl.java:174)
    at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:183)
    at java.base/sun.net.[www.http.HttpClient.openServer(HttpClient.java:532)](http://www.http.HttpClient.openServer(HttpClient.java:532))
    at java.base/sun.net.[www.http.HttpClient.openServer(HttpClient.java:637)](http://www.http.HttpClient.openServer(HttpClient.java:637))
    at java.base/sun.net.[www.protocol.https.HttpsClient](http://www.protocol.https.httpsclient/).<init>(HttpsClient.java:266)
    at java.base/sun.net.[www.protocol.https.HttpsClient.New(HttpsClient.java:380)](http://www.protocol.https.HttpsClient.New(HttpsClient.java:380))
    at java.base/sun.net.[www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:193)](http://www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:193))
    at java.base/sun.net.[www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1242)](http://www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1242))
    at java.base/sun.net.[www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1128)](http://www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1128))
    at java.base/sun.net.[www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:179)](http://www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:179))
    at java.base/sun.net.[www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1665)](http://www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1665))
    at java.base/sun.net.[www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1589)](http://www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1589))
    at java.base/sun.net.[www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:224)](http://www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:224))
    at io.openliberty.microprofile.metrics.internal.fat.DefaultHistogramBucketsTest.getHttpsServlet(DefaultHistogramBucketsTest.java:596)
    at io.openliberty.microprofile.metrics.internal.fat.DefaultHistogramBucketsTest.testDefaultBucketsEnabledHistogram(DefaultHistogramBucketsTest.java:102)
    at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
    at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:203)
    at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:363)
    at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:177)
```

**Comment by fmhwong** [comment]
@Channyboy This happened on Nov 8.  Not sure this test was run before or after your fix in https://github.com/OpenLiberty/open-liberty/issues/26779.

**Comment by Channyboy** [comment]
Fix for #26779 is already in when the failures occured.

But it seems that the secured port never starts up, so the https requests to `/metrics` fail.
